vector/src/main/java/im/vector/app/features/settings/devices/v2/othersessions/OtherSessionsViewModel.kt

```diff
@@ -85,6 +99,17 @@ class OtherSessionsViewModel @AssistedInject constructor(
             OtherSessionsAction.DeselectAll -> handleDeselectAll()
             OtherSessionsAction.SelectAll -> handleSelectAll()
             OtherSessionsAction.MultiSignout -> handleMultiSignout()
+            OtherSessionsAction.ToggleIpAddressVisibility -> handleToggleIpAddressVisibility()
+        }
+    }
+
+    private fun handleToggleIpAddressVisibility() = withState { state ->
+        val isShowingIpAddress = state.isShowingIpAddress
```

Comment:
Same remark for adding a reusable usecase.

Reply:
Refactored.